text-utils/ul.c

```diff
@@ -71,8 +71,8 @@
 #define	FREV	'7'
 
 enum {
-	NORMAL 	= 0,		/* Must be zero, see initbuf() */
-	ALTSET 	= 1 << 0,	/* Reverse */
+	G0_SET 	= 0,		/* Must be zero, see initbuf() */
+	G1_SET 	= 1 << 0,	/* Reverse */
```

Comment:
Frankly, the new names are not too readable for me, would be possible to get something more human friendly? ;-)

Reply:
I changed that commit to be hopefully better 580c63259bb1b59ec6730a1c4246269d76cb13e2.